lms/static/js/spec/learner_dashboard/course_enroll_view_spec.js

```diff
@@ -0,0 +1,61 @@
+define([
+        'backbone',
+        'jquery',
+        'js/learner_dashboard/models/course_card_model',
+        'js/learner_dashboard/views/course_enroll_view'
+    ], function (Backbone, $, CourseCardModel, CourseEnrollView) {
+        
+        'use strict';
+        
+        describe('Course Enroll View', function () {
+            var view = null,
+                courseCardModel,
+                setupView,
+                context = {      
+                    display_name: 'Astrophysics: Exploring Exoplanets',
+                    key: 'ANU-ASTRO1x',
+                    organization: {
+                        display_name: 'Australian National University',
+                        key: 'ANUx'
+                    },
+                    run_modes: [{
+                        start_date: 'Apr 25, 2016',
+                        end_date: 'Jun 13, 2016',
+                        course_key: 'course-v1:ANUx+ANU-ASTRO1x+3T2015',
+                        course_url: 'http://localhost:8000/courses/course-v1:edX+DemoX+Demo_Course/info',
+                        course_image_url: 'http://test.com/image1',
+                        marketing_url: 'http://test.com/image2',
+                        mode_slug: 'verified',
+                        run_key: '2T2016',
+                        is_enrolled: 'enrolled',
+                        certificate_url: '',
+                    }]
+                };
+
+            setupView = function(){
+                setFixtures('<div class="enrollment-container"></div>');
+                courseCardModel = new CourseCardModel(context);
+                view = new CourseEnrollView({
+                    model: courseCardModel
+                });
+            };
+
+            beforeEach(function() {
+                setupView(null);
```

Comment:
Why are you passing null into the setupView function? It currently doesn't accept any parameters.